diff -Naur a/gio/Makefile.in b/gio/Makefile.in
--- a/gio/Makefile.in	2012-06-02 19:05:00.899901930 +0200
+++ b/gio/Makefile.in	2012-06-02 17:35:25.712073390 +0200
@@ -399,19 +399,26 @@
 am_gdbus_OBJECTS = gdbus-tool.$(OBJEXT)
 gdbus_OBJECTS = $(am_gdbus_OBJECTS)
 gdbus_DEPENDENCIES = libgio-2.0.la $(top_builddir)/glib/libglib-2.0.la \
-	$(top_builddir)/gobject/libgobject-2.0.la
+	$(top_builddir)/gobject/libgobject-2.0.la \
+	$(top_builddir)/gthread/libgthread-2.0.la \
+	$(top_builddir)/gmodule/libgmodule-2.0.la
 am_gio_querymodules_OBJECTS = gio-querymodules.$(OBJEXT)
 gio_querymodules_OBJECTS = $(am_gio_querymodules_OBJECTS)
 gio_querymodules_DEPENDENCIES = $(top_builddir)/glib/libglib-2.0.la \
 	$(top_builddir)/gobject/libgobject-2.0.la \
-	$(top_builddir)/gmodule/libgmodule-2.0.la libgio-2.0.la \
+	$(top_builddir)/gmodule/libgmodule-2.0.la \
+	$(top_builddir)/gthread/libgthread-2.0.la \
+	libgio-2.0.la \
 	$(am__DEPENDENCIES_2)
 am_glib_compile_resources_OBJECTS = gvdb-builder.$(OBJEXT) \
 	glib-compile-resources.$(OBJEXT)
 glib_compile_resources_OBJECTS = $(am_glib_compile_resources_OBJECTS)
 glib_compile_resources_DEPENDENCIES =  \
 	$(top_builddir)/glib/libglib-2.0.la \
-	$(top_builddir)/gobject/libgobject-2.0.la libgio-2.0.la
+	$(top_builddir)/gobject/libgobject-2.0.la \
+	$(top_builddir)/gthread/libgthread-2.0.la \
+	$(top_builddir)/gmodule/libgmodule-2.0.la \
+	libgio-2.0.la
 am_glib_compile_schemas_OBJECTS = gvdb-builder.$(OBJEXT) \
 	glib-compile-schemas.$(OBJEXT)
 glib_compile_schemas_OBJECTS = $(am_glib_compile_schemas_OBJECTS)
@@ -422,11 +429,16 @@
 gresource_DEPENDENCIES = libgio-2.0.la \
 	$(top_builddir)/glib/libglib-2.0.la \
 	$(top_builddir)/gobject/libgobject-2.0.la \
+	$(top_builddir)/gthread/libgthread-2.0.la \
+	$(top_builddir)/gmodule/libgmodule-2.0.la \
 	$(am__DEPENDENCIES_2)
 am_gsettings_OBJECTS = gsettings-tool.$(OBJEXT)
 gsettings_OBJECTS = $(am_gsettings_OBJECTS)
 gsettings_DEPENDENCIES = $(top_builddir)/glib/libglib-2.0.la \
-	$(top_builddir)/gobject/libgobject-2.0.la libgio-2.0.la
+	$(top_builddir)/gobject/libgobject-2.0.la \
+	$(top_builddir)/gthread/libgthread-2.0.la \
+	$(top_builddir)/gmodule/libgmodule-2.0.la \
+	libgio-2.0.la
 DEFAULT_INCLUDES = -I.@am__isrc@ -I$(top_builddir)
 depcomp = $(SHELL) $(top_srcdir)/depcomp
 am__depfiles_maybe = depfiles
@@ -1252,7 +1264,9 @@
 CLEANFILES = gdbus-daemon-generated.c gdbus-daemon-generated.h gio-public-headers.txt gconstructor_as_data.h
 glib_compile_resources_LDADD = \
 	$(top_builddir)/glib/libglib-2.0.la \
-	$(top_builddir)/gobject/libgobject-2.0.la	\
+	$(top_builddir)/gobject/libgobject-2.0.la \
+	$(top_builddir)/gthread/libgthread-2.0.la \
+	$(top_builddir)/gmodule/libgmodule-2.0.la \
 	libgio-2.0.la
 
 glib_compile_resources_SOURCES = \
@@ -1266,6 +1280,7 @@
 	$(top_builddir)/glib/libglib-2.0.la		\
 	$(top_builddir)/gobject/libgobject-2.0.la	\
 	$(top_builddir)/gmodule/libgmodule-2.0.la	\
+	$(top_builddir)/gthread/libgthread-2.0.la	\
 	libgio-2.0.la					\
 	$(NULL)
 
@@ -1280,6 +1295,8 @@
 gsettings_LDADD = \
 	$(top_builddir)/glib/libglib-2.0.la		\
 	$(top_builddir)/gobject/libgobject-2.0.la	\
+	$(top_builddir)/gthread/libgthread-2.0.la	\
+	$(top_builddir)/gmodule/libgmodule-2.0.la	\
 	libgio-2.0.la
 
 gsettings_SOURCES = gsettings-tool.c
@@ -1288,7 +1305,9 @@
 gdbus_SOURCES = gdbus-tool.c
 gdbus_LDADD = libgio-2.0.la \
 	$(top_builddir)/glib/libglib-2.0.la 		\
-	$(top_builddir)/gobject/libgobject-2.0.la
+	$(top_builddir)/gobject/libgobject-2.0.la	\
+	$(top_builddir)/gthread/libgthread-2.0.la	\
+	$(top_builddir)/gmodule/libgmodule-2.0.la
 
 completiondir = $(sysconfdir)/bash_completion.d
 completion_DATA = \
@@ -1301,6 +1320,8 @@
 gresource_LDADD = libgio-2.0.la \
 	$(top_builddir)/glib/libglib-2.0.la 		\
 	$(top_builddir)/gobject/libgobject-2.0.la \
+	$(top_builddir)/gthread/libgthread-2.0.la \
+	$(top_builddir)/gmodule/libgmodule-2.0.la \
 	$(LIBELF_LIBS)
 
 all: $(BUILT_SOURCES)
