diff -Naur a/src/journal/journalctl.c b/src/journal/journalctl.c
--- a/src/journal/journalctl.c	2013-03-08 10:23:25.528526904 +0100
+++ b/src/journal/journalctl.c	2013-03-08 10:23:42.856334353 +0100
@@ -728,7 +728,7 @@
         if (ioctl(fd, FS_IOC_GETFLAGS, &attr) < 0)
                 log_warning("FS_IOC_GETFLAGS failed: %m");
 
-        attr |= FS_SECRM_FL|FS_NODUMP_FL|FS_SYNC_FL|FS_NOCOW_FL;
+        attr |= FS_SECRM_FL|FS_NODUMP_FL|FS_SYNC_FL;
 
         if (ioctl(fd, FS_IOC_SETFLAGS, &attr) < 0)
                 log_warning("FS_IOC_SETFLAGS failed: %m");
