diff --git a/src/dircolors.h b/src/dircolors.h
index 74a4494..90f28e0 100644
--- a/src/dircolors.h
+++ b/src/dircolors.h
@@ -58,6 +58,7 @@ static char const G_line[] =
   'T','E','R','M',' ','s','t','-','2','5','6','c','o','l','o','r',0,
   'T','E','R','M',' ','t','e','r','m','i','n','a','t','o','r',0,
   'T','E','R','M',' ','v','t','1','0','0',0,
+  'T','E','R','M',' ','v','t','1','0','2',0,
   'T','E','R','M',' ','x','t','e','r','m',0,
   'T','E','R','M',' ','x','t','e','r','m','-','1','6','c','o','l','o','r',0,
   'T','E','R','M',' ','x','t','e','r','m','-','2','5','6','c','o','l','o','r',0,
